sources/imperative/reader/Reader.swift

```diff
@@ -52,13 +52,22 @@ public final class CSVReader: IteratorProtocol, Sequence {
       guard !headers.isEmpty else { throw Error._invalidEmptyHeader() }
       self.headers = headers
       self.count = (rows: 1, fields: headers.count)
+    case let .lineNumber(index):
+        // Parse rows, but ignore them
+        for _ in 0..<index {
+            _ = try self._parseLine(rowIndex: 0)
+        }
+        guard let headers = try self._parseLine(rowIndex: index) else { self.status = .finished; return }
+        guard !headers.isEmpty else { throw Error._invalidEmptyHeader() }
+        self.headers = headers
+        self.count = (rows: index, fields: headers.count)
```

Comment:
For a similar reason to the previous note, this should be `self.count = (rows: 0, fields: headers.count)`.